packages/libp2p/src/connection-manager/index.ts

```diff
@@ -257,6 +257,7 @@ export class DefaultConnectionManager implements ConnectionManager, Startable {
       addressSorter: init.addressSorter ?? publicAddressesFirst,
       maxParallelDials: init.maxParallelDials ?? MAX_PARALLEL_DIALS,
       maxPeerAddrsToDial: init.maxPeerAddrsToDial ?? MAX_PEER_ADDRS_TO_DIAL,
+      maxParallelDialsPerPeer: init.maxParallelDialsPerPeer ?? MAX_PARALLEL_DIALS_PER_PEER,
```

Comment:
Where is this value being used?

Reply:
https://github.com/libp2p/js-libp2p/blob/master/packages/libp2p/src/connection-manager/dial-queue.ts#L428